app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -98,6 +99,12 @@ class SuggestedEditsTasksFragment : Fragment() {
         clearContents()
     }
 
+    private fun Group.addOnClickListener(listener: (view: View) -> Unit) {
+        referencedIds.forEach { id ->
+            rootView.findViewById<View>(id).setOnClickListener(listener)
```

Comment:
The `rootView` can be replaced by `userStatsClickTarget` to reduce unnecessary search to the entire layout.